### PR TITLE
alts: annotate ALTS public API

### PR DESCRIPTION
**Pull request by jiangtaoli2016**

### CODE REVIEW

File: alts/src/main/java/io/grpc/alts/AltsChannelBuilder.java [hunk 1]
```diff
@@ -45,6 +46,7 @@
  * ALTS version of {@code ManagedChannelBuilder}. This class sets up a secure and authenticated
  * commmunication between two cloud VMs using ALTS.
  */
+@ExperimentalApi
```
Comment (carl-mastrangelo) [inline alts/src/main/java/io/grpc/alts/AltsChannelBuilder.java]: We typically make a tracking issue for each usage of ExperimentalApi,    You can see it in other usages of the annotation.  It allows us to keep track of when to make it non experimental.

File: alts/src/main/java/io/grpc/alts/transportsecurity/AltsAuthContext.java [hunk 2]
```diff
@@ -17,12 +17,14 @@
...
 import com.google.common.annotations.VisibleForTesting;
+import io.grpc.Internal;
 import io.grpc.alts.Altscontext.AltsContext;
 import io.grpc.alts.Handshaker.HandshakerResult;
 import io.grpc.alts.TransportSecurityCommon.RpcProtocolVersions;
 import io.grpc.alts.TransportSecurityCommon.SecurityLevel;
 
 /** AltsAuthContext contains security-related context information about an ALTs connection. */
+@Internal
 public final class AltsAuthContext {
```
Comment (carl-mastrangelo) [inline alts/src/main/java/io/grpc/alts/transportsecurity/AltsAuthContext.java]: If this is actually internal, you should make the class package private, and using an Internal accessor.   See some of the classes named Internal* in the core repo.

File: alts/src/main/java/io/grpc/alts/AltsChannelBuilder.java (hunk 1 above)
Comment (jiangtaoli2016) [reply alts/src/main/java/io/grpc/alts/AltsChannelBuilder.java]: Added a tracking issue.

File: alts/src/main/java/io/grpc/alts/transportsecurity/AltsAuthContext.java (hunk 2 above)
Comment (jiangtaoli2016) [reply alts/src/main/java/io/grpc/alts/transportsecurity/AltsAuthContext.java]: AltsAuthContext and TsiPeer are not used today, but will be used in the future for authorization check, thus they are public. let me update them as ExperimentalApi

**Comment by jiangtaoli2016** [comment]
@carl-mastrangelo I have revised the code. Not sure whether I need to rename the Internal Public Class. Let me send an email to discuss.

**Pull request closed without merging**